Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -1,15 +1,114 @@
-﻿namespace BootCamp.Chapter1
+﻿using System;
+namespace BootCamp.Chapter1
 {
     public static class ArrayOperations
     {
+        // *************private helpers**********
         /// <summary>
+        /// Test that Array is neither null or empty.
+        /// </summary>
+        /// <returns>True if array can be manipulated by standard functions.</returns>
+        /// 
+        private static bool ValidateArrayForManipulation(int[] array)
+        {
+            return ((array != null) && (array.Length != 0));
+        }
+
+        /// <summary>
+        /// Makes the array one index item shorter.
+        /// </summary>
+        /// <returns>A new array with (original length - 1)</returns>
+        /// 
+        private static int[] MakeAShorterArray(int[] array)
+        {
+            return new int[array.Length - 1];
+        }
+
+        /// <summary>
+        /// Makes a new array with one index with an int
+        /// </summary>
+        /// <returns>A new array with a single index with value {number}>
+        /// 
+        private static int[] AddOneNumberToNullArray(int number)
+        {
+            return new[] { number };
+        }
+
+         /// <summary>
+        /// Makes the array one index item longer.
+        /// </summary>
+        /// <returns>A new array with (original length+ 1)</returns>
+        private static int[] MakeALongerArray(int[] array)
+        {
+            return new int[array.Length + 1];
+        }
+
+        /// <summary>
+        /// set a new value at either the very first or very last index point
+        /// </summary>
+        /// <returns>array that has one value replaced at specified index
+        /// 
+
+        private static void PlaceTheNewNumberAtTheAppropriateIndexPosition(int[] lengthenedArray, string position, int number)
+        {
+            switch (position)
+            {
+                case "Last":
+                    lengthenedArray[lengthenedArray.Length - 1] = number;
+                    break;
+
+                case "First":
+                    lengthenedArray[0] = number;
+                    break;
+
+                default:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty 'default' clause.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887772)